docs/content/users/install/ddev-installation.md

```diff
@@ -43,22 +43,18 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     ### Debian/Ubuntu
 
-    DDEV’s Debian and RPM packages work with `apt` and `yum` repositories and most variants that use them, including Windows WSL2:
+    DDEV’s Debian and RPM packages work with `apt` and `yum` repositories and most variants that use them, including Windows WSL2. Make sure you have cURL installed:
```

Comment:
Curl is not included in many standard Ubuntu and Debian distros. You'd be surprised :)

Reply:
Why we added curl:

- #5915